web/templates/pages/member_page.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html' %}
+
+{% load staticfiles %}
+
+{% block title %}TrackCat Member{% endblock %}
+
+{% block content%}
+
+	<div class="row">
+		<div class="member-profile">
+			<div class="col-md-12">
+			<h2>{% if user.fullname %}
+					{{ user.fullname }} 
+				{% else %}
+					{{ user.username }}
+				{% endif %}'s profile</h2>
+			</div>	
+		</div>
+	</div>
+	<div class="row">
+		<div class="col-md-4 col-xs-6">
+			<img src="../../static/img/{{user.profile_picture}}">
+			<p><strong>GitHub Name:</strong></p>
+			<p><a href="{{ user.githubuser}}"> {{ user.githubuser }}</a></p>
```

Comment:
Is `{{ user.githubuser }}` even a link? I am sure it is not, so this will lead to nowhere.

And also, using a space on at the beginning and no at the end...